components/health_controller/src/reporters.rs

```diff
@@ -92,11 +113,23 @@ impl UnifiedSlowScore {
         &mut self.factors[factor as usize]
     }
 
-    // Returns the maximum score of all factors.
-    pub fn get_score(&self) -> f64 {
+    // Returns the maximum score of disk factors.
+    pub fn get_disk_score(&self) -> f64 {
+        self.factors
+            .iter()
+            .enumerate()
+            .filter(|(idx, _)| *idx < InspectFactor::Network as usize)
```

Comment:
Please add some necessary annotations here, i.e. "the factor `InspectFactor::Network` is the final element when initializing the `factors` list, where other factors related to `disk-io` are placed ahead of it."